content/graphing/infrastructure/_index.md

```diff
@@ -100,9 +100,7 @@ An easy way to accomplish this would be to use the following script that leverag
 
 This script outputs all the current running Agents and their version numbers to a separate document.  Additionally, you can edit the script to input a desired Version number if you would also like all the running Agents that are under a particular version number.  There is also a separate file if you would like to convert the JSON output into a CSV file for your review.
 
-Once you determine which hosts you would like to update you can either manually install the Agent from the [install page](https://app.datadoghq.com/account/settings#agent).   
-
-Or you can make use of one our automation integrations like [Chef](/integrations/chef), [Puppet](/integrations/puppet), or [Ansible](/integrations/ansible).
+Once you determine which hosts you would like to update you can either manually install the Agent from the [install page](https://app.datadoghq.com/account/settings#agent), or you can make use of one of our automation integrations like [Chef](/integrations/chef), [Puppet](/integrations/puppet), or [Ansible](/integrations/ansible).
```

Comment:
s/or you can make use of one of our automation/or you can use one of our automation/